src/csharp/Microsoft.Spark/Microsoft.Spark.csproj

```diff
@@ -6,12 +6,15 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IsPackable>true</IsPackable>
-
     <Description>.NET for Apache Spark</Description>
     <PackageReleaseNotes>https://github.com/dotnet/spark/tree/master/docs/release-notes</PackageReleaseNotes>
     <PackageTags>spark;dotnet;csharp</PackageTags>
   </PropertyGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.1' ">
```

Comment:
@ahsonkhan @marcusturewicz Thanks for doing this. However, JSON parsing is not in a critical path, and do you think it's worthwhile to add an extra complexity for perf? How about getting the first version in with minimal changes and iterate on it if necessary? What do you guys think?

Reply:
I don't have perf insights around usage of the component being updated here, so I would defer to the judgement of folks like yourself who own this space.

I would imagine cross-compiling would be important to take advantage of the latest improvements within the framework, at least for areas that are perf critical, so this helps set the precedent for it. If this is introducing extra complexity without clear benefit, then it makes sense to defer such changes.

> How about getting the first version in with minimal changes and iterate on it if necessary?

That seems like a viable way to make progress and iterate where we separate the porting from perf improvements. It makes sense to me.

> However, JSON parsing is not in a critical path, and do you think it's worthwhile to add an extra complexity for perf?

Mainly out of curiosity, can you expand on why JSON parsing is not perf critical for spark scenarios (for my edification)?